maquetado/scss/styles.scss

```diff
@@ -1,3 +1,32 @@
-.title {
-  color: red;
+.bold {
+    font-weight: bold;
+}
+
+.book-title {
+    font-size: 32px;
+    margin-bottom: 20px;
```

Comment:
If you have to use two margin properties, prefer using the shortcut `margin`: https://www.w3schools.com/css/css_margin.asp

This applies to all css properties that have shortcuts, such as `border`, `padding`, `flex`, etc.